scripts/launch-website.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+(sleep 10;
```

Comment:
Could you please add a comment line above this to explain the reason for the `sleep` and the subquery `&`?